src/pages/reference/transforms/filter-schema.md

```diff
@@ -110,7 +110,7 @@ type User {
 ```
 <InlineAlert variant="info" slots="text"/>
 
-For information about "bare" and "wrap" modes, please read the [dedicated section](transforms-introduction.md#two-different-modes).
+For information about "bare" and "wrap" modes, please read the [dedicated section](index.md#two-different-modes).
```

Comment:
This link doesn't work. I'm guessing you'll have to write the link as `./index.md#two-different-modes`.

While you're editing this line, get rid of the "please".